src/index.html

```diff
@@ -5,8 +5,338 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#home">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+
+          <li class="header__nav-item">
+            <a class="is-active" href="#Apple">Apple</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#Samsung">Samsung</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#Smartphones">Smartphones</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#Laptops & Computers"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#Gadgets">Gadgets</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#Tablets">Tablets</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#Photo">Photo</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#Video">Video</a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards">
+      <article class="cards__card" data-qa="card">
```

Comment:
also, let's create a separate bem block for card, it will make our code more reusable